src/main/java/org/zone/region/flag/entity/player/move/message/leaving/LeavingFlagType.java

```diff
@@ -34,23 +37,44 @@
 
     @Override
     public @NotNull LeavingFlag load(@NotNull ConfigurationNode node) throws IOException {
-        String message = node.node("Message").getString();
+        String message = node.node("LeavingMessage").getString();
+        String displayType = node.node("LeavingMessageDisplayType").getString();
         if (message == null) {
             throw new IOException("Cannot read message");
         }
+        if (displayType == null) {
+            throw new IOException("Could not get the display type");
```

Comment:
Cannot punish 1.0.0 users for using the plugin. Find a better way to handle this